content/docs/command-reference/run.md

```diff
@@ -125,22 +122,44 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
-  regular output). See `dvc metrics` to learn more about using metrics.
-
-- `-M <path>`, `--metrics-no-cache <path>` - the same as `-m` except that files
-  are not tracked by DVC. It means that they are not cached, and it's up to a
-  user to save and version control them. This is typically desirable with metric
-  files, because they are small enough to be tracked by Git directly. See also
-  the difference between `-o` and `-O`.
-
-- `-f <filename>`, `--file <filename>` - specify a path and/or file name for the
-  stage file generated by this command (e.g. `-f stages/stage.dvc`). This
-  overrides the default file name: `<file>.dvc`, where `<file>` is the file name
-  of the first output or metric.
+- `-p [<path>:]<params_list>`, `--params [<path>:]<params_list>` - specify a set
+  of [parameter dependencies](/doc/command-reference/params) the stage depends
+  on, from a parameters file. This is done by sending a coma separated list as
+  argument, e.g. `-p learning_rate,epochs`. The default parameters file name is
+  `params.yaml`, but this can be redefined with a prefix in the argument sent to
+  this option, e.g. `-p parse_params.yaml:threshold`. See `dvc params` to learn
+  more about parameters.
+
+- `-m <path>`, `--metrics <path>` - specify a metrics file produces by this
+  stage. This option behaves like `-o` but registers the file in a `metrics`
+  field inside the `dvc.yaml` stage. Metrics are usually small, human readable
+  files (JSON or YAML) with scalar numbers or other simple information that
+  describes a model (or any other data artifact). See `dvc metrics` to learn
+  more about _metrics_.
+
+- `-M <path>`, `--metrics-no-cache <path>` - the same as `-m` except that DVC
+  does not track the metrics file. This means that the file is not cached, so
+  it's up to the user to save and version control it. This is typically
+  desirable with _metrics_ because they are small enough to be tracked with Git
+  directly. See also the difference between `-o` and `-O`.
+
+- `--plots <path>` - specify a plot metrics file produces by this stage. This
+  option behaves like `-o` but registers the file in a `metrics` field inside
+  the `dvc.yaml` stage. Plot metrics are data series stored in tabular (CSV or
+  TSV) or hierarchical (JSON or YAML) files, with complex information that
+  describes a model (or any other data artifact). See `dvc plots` to learn more
+  about plots.
+
+- `--plots-no-cache <path>` - the same as `--plots` except that DVC does not
+  track the plots metrics file. This means that the file is not cached, so it's
+  up to the user to save and version control it. See also the difference between
+  `-o` and `-O`.
+
+- `--external` - allow outputs that are outside of the DVC repository. See
+  [Managing External Data](/doc/user-guide/managing-external-data).
+
+- `--file <path>` - specify name of the YAML file this command will generate
+  instead of the default, `dvc.yaml`.
```

Comment:
This is ignored for `dvc.yaml`. We don't have support for custom `dvc.yaml` files.

Reply:
OK thanks. How do you create a dvc.yaml file inside a subdirectory?

Reply:
Not supported at this point. `dvc.yaml` is created at the location where `dvc run` was run from.

Reply:
OK. So multiple `dvc.yaml` files are supported, you just move to another directory and use `dvc run` from there, right?

Reply:
I checked this and it is the case. Confirmed